tests/video/test_loading_cursor.py

```diff
@@ -0,0 +1,27 @@
+from panda3d.core import MovieVideo
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = MovieVideo.get(movie_path)
+    return getFile #Obtains a MovieVideo that references a file
+
+@pytest.fixture
```

Comment:
I doesn't really matter for one file, but fixtures should probably go in `tests/video/conftest.py`